_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,341 @@
+  any last-minute problems before the final release of this maintenance
+  update.
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][core commits],
+[LND][lnd commits], [C-lightning][cl commits], and [libsecp256k1][secp
+commits].*
+
+{% include linkers/github-log.md
+  refname="core commits"
+  repo="bitcoin/bitcoin"
+  start="e70a19e7132dac91b7948fcbfac086f86fec3d88"
+  end="35739976c1d9ad250ece573980c57e7e7976ae23"
+%}
+{% include linkers/github-log.md
+  refname="lnd commits"
+  repo="lightningnetwork/lnd"
+  start="d4b042dc1946ece8b60d538ade8e912f035612fe"
+  end="4da1c867c3209dab4e4a824b73d89fc38b616b37"
+%}
+{% include linkers/github-log.md
+  refname="cl commits"
+  repo="ElementsProject/lightning"
+  start="62e6a9ff542e40364b67a7aa419e33ed72b96a42"
+  end="d5aaa11373cc6759f9f894a1daf7fb88d0834bc9"
+%}
+{% include linkers/github-log.md
+  refname="secp commits"
+  repo="bitcoin-core/secp256k1"
+  start="1086fda4c1975d0cad8d3cad96794a64ec12dca4"
+  end="314a61d72474aa29ff4afba8472553ad91d88e9d"
+%}
+
+- [C-Lightning #2075][] adds support for plugins.  As their
+  [documentation][cl plugin] describes, "plugins are a simple yet
+  powerful way to extend the functionality provided by c-lightning.
+  They are subprocesses that are started by the main `lightningd` daemon
+  and can interact with `lightningd` in a variety of ways."  At present,
+  plugins can add command-line options to the main process, but there
+  are plans to allow them to add new JSON-RPC commands, receive events,
+  and insert code to be called by hooks in the main process.  A
+  `helloworld` plugin written in Python is provided with C-Lightning as
+  an example.
+
+- [Bitcoin Core #14411][] The [listtransactions][rpc listtransactions]
+  has its filter parameter partly restored, making it possible to
+  retrieve a list of the transactions sent to addresses or scripts with
+  a particular label.  This has been backported to the 0.17 branch as
+  [PR #14441][Bitcoin Core #14441] and is expected to be distributed in
+  the next maintenance release.
+
+- [LND #2124][] adds another essential portion of watchtower support,
+  specifically the ability to detect that an attacker has made an
+  onchain attempt steal from one of the watchtower's users.  The
+  watchtower can use the information from the onchain transaction to
+  decrypt a breach remedy transaction previously provided by the victim
+  in order to both negate the attack and penalize the attacker by
+  claiming any funds the attacker legitimately owned in the channel.  In
+  the current implementation, the watchtower receives a percentage of
+  the recovered funds to compensate it for its diligent monitoring.
+  This merge is an extension of PRs [#1535][LND #1535] and [#1512][LND
+  #1512] described in [Newsletter #19][] and is a major step towards
+  making LN safer for everyday users.
+
+## Special thanks
+
+We thank Chris Decker, PracticalSwift, and Rene Pickhardt for providing
```

Comment:
Christian Decker, practicalswift, and René Pickhardt

Reply:
D'oh.  Thanks!